jetstream/max_text/README.md

```diff
@@ -0,0 +1,289 @@
+# JetStream MaxText Inference on v5e Cloud TPU VM User Guide
```

Comment:
This file should really be in MaxText repo, similar to pytorch one.

Reply:
> This file should really be in MaxText repo, similar to pytorch one.

We can do it later (after separation maxtext training vs inference ), there is no place I can put in maxtext repo right now.

Reply:
Sounds good.